test/cmdlineTests/evmasm_transient_storage_state_variable/args

```diff
@@ -0,0 +1 @@
+--evm-version cancun --no-cbor-metadata --optimize --asm --debug-info none
```

Comment:
I guess I thought other versions could be tested as well so that it was required to fix a version.
Removed that.

Reply:
Looking at it again, looks like it was just copied from another test? And I think that back when you added those tests `cancun` wasn't the default yet which is why it was necessary?

If that's the case then it may be a good idea to look through the existing CLI tests for transient storage, see if we have it in any others and remove it. It's not wrong, but we're better off without it.